test/integration/targets/docker_swarm/tasks/main.yml

```
@@ -13,11 +13,16 @@
     diff: no
     ignore_errors: yes
 
+  - name: Kill docker daemon
+    command: systemctl stop -s 9 docker
```

Ah, I think I misread the `systemctl` documentation; I think signals can only be specified for `systemctl kill`. Can you try the following?
```suggestion
    command: systemctl kill -s 9 docker
```